### PR TITLE
RDAPI-19315 FIPS update

### PR DESCRIPTION
**Pull request by alexearnshaw**
## Describe the changes

* Add info about OpenSSL and FIPS in Mar20 RN
* Add note about FIPS support to main FIPS topic
* Remove continuous updates notes from both RNs

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is clear and complete
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [ ] Follows [best practices](https://techweb.axway.com/confluence/display/RDAPI/Best+practices+for+developer+documentation)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links
* [ ] Is discoverable and navigable

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [x] You have applied a size label (unless this work is tracked in the ReadMeFirst JIRA project)
* [x] You have applied a production label if this needs a manual push to Zoomin
* [x] You have added it to the correct GitHub project
* [x] You have added it to the correct Sprint milestone
* [x] You have appended a `fixes` line to this description if this PR fixes a specific GitHub issue
* [x] You have added a new collection to Netlify CMS config (static/admin/config.js) if required (new doc sections)
* [x] You have updated the Zoomin classification file if required (new AMPC topics)

### CODE REVIEW

File: content/en/docs/apim_administration/apigtw_admin/admin_fips.md [hunk 1]
```diff
@@ -6,6 +6,10 @@
 "description": "Run an API Gateway instance or a Policy Studio client in FIPS mode."
 }
 
+{{< alert title="Note" color="primary" >}}
+Running API Gateway in FIPS mode is not supported in API Gateway 7.7 Mar20 update or later, as the Mar20 update uses OpenSSL 1.1.1, which does not support FIPS.
```
Comment (andreamussap) [inline content/en/docs/apim_administration/apigtw_admin/admin_fips.md]: Suggestion to shorten and remove repetition - 

Running API Gateway in FIPS mode is not supported in API Gateway 7.7 Mar20 update or later as this release uses OpenSSL 1.1.1, which does not support FIPS.

File: content/en/docs/apim_relnotes/20200330_apip_relnotes/_index.md [hunk 2]
```diff
@@ -7,10 +7,6 @@ date: 2020-03-11T00:00:00.000Z
 description: Learn about the new features and enhancements in this release of API Portal.
 ---
 
-{{< alert title="Note" color="primary" >}}
-This release note will be continuously updated until the release date.
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20200330_apip_relnotes/_index.md]: why did you remove this?

File: content/en/docs/apim_administration/apigtw_admin/admin_fips.md (hunk 1 above)
Comment (alexearnshaw) [reply content/en/docs/apim_administration/apigtw_admin/admin_fips.md]: I had that originally, but decided to be more specific, as using "this release" might be confusing and make people think all 7.7 releases use 1.1.1 which is not true?

File: content/en/docs/apim_relnotes/20200330_apip_relnotes/_index.md (hunk 2 above)
Comment (alexearnshaw) [reply content/en/docs/apim_relnotes/20200330_apip_relnotes/_index.md]: I removed it in both RN as we only added this when we planned to have these visible on zoomin, but now we are not showing them on Zoomin so the notes are not needed?

File: content/en/docs/apim_relnotes/20200330_apimgr_relnotes/_index.md [hunk 3]
```diff
@@ -35,6 +31,14 @@ It is important, especially when upgrading from an earlier version, to be aware
 
 API Gateway and API Manager 7.7 and later support OpenJDK JRE, and this update includes Zulu OpenJDK 1.8 JRE instead of Oracle JRE 1.8.
 
+### OpenSSL and FIPS support
+
+In this update OpenSSL has been upgraded to OpenSSL 1.1.1.
+
+As OpenSSL 1.1.1 does not support FIPS, running API Gateway in FIPS mode is not supported in this update. FIPS support will be available in a future update.
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20200330_apimgr_relnotes/_index.md]: you can remove this, it's better stated already on the following sentence, so no need for repetition

File: content/en/docs/apim_relnotes/20200330_apimgr_relnotes/_index.md [hunk 4]
```diff
@@ -35,6 +31,14 @@ It is important, especially when upgrading from an earlier version, to be aware
...
+
+References to FIPS in the documentation have not been removed as FIPS support will be available again in the future, however, this does not mean that FIPS is still supported, and the references should be ignored.
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20200330_apimgr_relnotes/_index.md]: again in the **future; however**, this does not mean

File: content/en/docs/apim_administration/apigtw_admin/admin_fips.md (hunk 1 above)
Comment (andreamussap) [reply content/en/docs/apim_administration/apigtw_admin/admin_fips.md]: sorry I don't get you:
"as **the Mar20 update** uses OpenSSL 1.1.1"
is the same as
"as **this release** uses OpenSSL 1.1.1"
IMHO ??

Comment (alexearnshaw) [reply content/en/docs/apim_administration/apigtw_admin/admin_fips.md]: This FIPS topic is part of the main 7.7 docs, not the Mar20 RN, so saying "this release" here could mean any 7.7 release (including 7.7 2019 or 7.7 Jan20) which would not be correct?
